### DISCUSSION TITLE
Show off your stuff!

### DISCUSSION THREAD

**Discussion by Griatch** [evennia/evennia#2270] [My Evennia Project]
This category is a place for showing off your work - feel free to post your progress, ask for feedback or announce your future awesome Evennia game. You can also use a thread as your 'dev blog' if you want. :smile: 

[Click here to start a new thread](https://github.com/evennia/evennia/discussions/new) (pick `Community Projects and Ideas` as category).

PS: Don't forget to add you game to the [Evennia game index](http://games.evennia.com)! You can add your game even if it's unfinished or just started (just mark it as `pre-alpha`). Make us excited about what's to come!

**Comment by duysqubix** [comment]
Awesome! Will update soon with information on the Unofficial Elder Scrolls Mud I am working on.  Much of the core functionality is in place.
